src/encoded/static/components/regulome_search.js

```diff
@@ -580,8 +580,8 @@ export class RegulomeSearch extends React.Component {
         // Bind this to non-React methods.
         this.requests = [];
         this.onFilter = this.onFilter.bind(this);
-        this.loadHg19ValisData = this.loadHg19ValisData.bind(this);
-        this.loadGRCh38ValisData = this.loadGRCh38ValisData.bind(this);
+        //this.loadHg19ValisData = this.loadHg19ValisData.bind(this);
```

Comment:
Nice to delete code instead of commenting it out, but for Regulome, probably not such a big deal.